docs/source/index.rst

```diff
@@ -67,9 +72,11 @@ When the :py:meth:`.fit() <cleanlab.classification.CleanLearning.fit>` method is
    from sklearn.linear_model import LogisticRegression
    from cleanlab.classification import CleanLearning
 
-   clf = LogisticRegression() # any classifier implementing the sklearn API
-   cl = CleanLearning(clf=clf)
-   cl.fit(X=X, labels=labels)
+   # Train an ML model with noisy labels - 3 lines of code
```

Comment:
if you're trying to save space, this comment may not be needed given it's the section title above this